config/azure/build.yaml

```diff
@@ -43,15 +54,32 @@ jobs:
      displayName: 'Install Toolchain (Darwin)'
      condition: eq( variables['Agent.OS'], 'Darwin' )
 
+   - script: g++ --version
+     condition: or( eq( variables['Agent.OS'], 'Darwin' ), eq( variables['Agent.OS'], 'Linux' ) )
```

Comment:
Be good to shove this down at line 74 to save adding another task - it was just to provide a sanity-check informational print out I think.

Reply:
Taken care of in 0fdd8a4